.github/workflows/php7.2.yml

```diff
@@ -0,0 +1,98 @@
+name: "Website build"
+
+on: ["push", "pull_request"]
+
+env:
+  PHP_VERSION: "7.2"
+
+jobs:
+  tests:
+
+    runs-on: "ubuntu-16.04"
+
+    services:
+      mysql:
+        image: "mysql:5.7"
+        env:
+          MYSQL_DATABASE: "doctrine_website_test"
+          MYSQL_ROOT_PASSWORD: "VdtLtifRh4gt37T"
+        ports:
+          - "3306:3306"
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
+
+    steps:
+    - uses: "actions/checkout@v2"
+
+    - name: "Cache Composer packages"
+      id: "composer-cache"
+      uses: "actions/cache@v2"
+      with:
+        path: "vendor"
+        key: "${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}"
+        restore-keys: |
+          ${{ runner.os }}-php-
+
+    - name: "Setup PHP Action"
+      uses: "shivammathur/setup-php@v2"
+      with:
+        php-version: "${{ env.PHP_VERSION }}"
+        coverage: "none"
+
+    - name: "Install PHP dependencies"
+      if: "steps.composer-cache.outputs.cache-hit != 'true'"
+      run: "composer install --prefer-dist --no-progress --no-suggest"
+
+    - name: "apt-get install yarn"
+      run: "sudo apt-get update && sudo apt-get install -y yarn"
+
+    - name: "yarn install"
+      run: "yarn install"
+
+    - name: "Run tests"
+      run: "./vendor/bin/phpunit"
+
+    - name: "Setup PHP Action"
+      uses: "shivammathur/setup-php@v2"
+      with:
+        php-version: "${{ env.PHP_VERSION }}"
+        coverage: "xdebug"
+
+    - name: "Build website"
+      run: "./bin/console --env=test -v build-all"
+
+    - name: "Run tests with coverage"
+      run: "./vendor/bin/phpunit --coverage-clover clover.xml"
+
+    - name: "Deliver code coverage"
+      run: "bash <(curl -s https://codecov.io/bash)"
+
+  code-analysis:
+
+    runs-on: "ubuntu-16.04"
+
+    steps:
+    - uses: "actions/checkout@v2"
+
+    - name: "Cache Composer packages"
+      id: "composer-cache"
+      uses: "actions/cache@v2"
+      with:
+        path: "vendor"
+        key: "${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}"
+        restore-keys: |
+          ${{ runner.os }}-php-
+
+    - name: "Setup PHP Action"
+      uses: "shivammathur/setup-php@v2"
+      with:
+        php-version: "${{ env.PHP_VERSION }}"
+
+    - name: "Install dependencies"
+      if: "steps.composer-cache.outputs.cache-hit != 'true'"
+      run: "composer install --prefer-dist --no-progress --no-suggest"
+
+    - name: "Run Code Sniffer"
+      run: "./vendor/bin/phpcs"
```

Comment:
Let's reuse https://github.com/doctrine/.github/blob/main/workflow-templates/coding-standards.yml ? I don't seem to have the correct permissions to see the "new workflow" button after clicking "actions" but maybe you do?

Reply:
@greg0ire I can. As far as I understand I need to create a new workflow "Coding standards" (by Doctrine) in a new branch. I assume that this new branch will be merged into this one, right? Or do you expect that the new branch will be merged in `master` and the rebased in this branch?

Reply:
I guess you gave me the answer I was looking for :smile: